src/main/java/core/basesyntax/service/activity/BalanceActivityService.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.service.activity;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.service.FruitServiceImpl;
+import core.basesyntax.service.GetListOfActivity;
+import core.basesyntax.service.GetListOfActivityImpl;
+import java.util.List;
+
+public class BalanceActivityService implements ActivityService {
+    private static final int FRUIT_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
+    private static final String ACTIVITY = "b";
```

Comment:
it's better to store such values in enum, look at the example in the description to this task